code/game/machinery/doors/windowdoor.dm

```diff
@@ -301,11 +302,11 @@
 	try_to_activate_door(null, user)
 
 /obj/machinery/door/window/try_to_activate_door(obj/item/I, mob/user)
-	if (..())
+	if(..())
 		autoclose = FALSE
 
 /obj/machinery/door/window/try_to_crowbar(obj/item/I, mob/user)
-	if(!hasPower())
+	if(!hasPower() || HAS_TRAIT(I, TRAIT_DOOR_PRYER))
```

Comment:
Door prying for regular airlocks has a do_after that takes like 5 seconds, it should do the same